src/qibo/backends/tensorflow.py

```diff
@@ -47,7 +47,8 @@ def __init__(self):
         self.op = None
         if op._custom_operators_loaded:
             self.op = op
-
+        self._seed = 1234 # seed to use in the measurement frequency custom op
```

Comment:
Could we set the current seed used by tf instead?

Reply:
That was my original plan, however I could not find a "get_seed" method for Tensorflow. I will have a second look though. An alternative solution would be to set the seed randomly, for example via time, and just set the same seed for Tensorflow, like
```Python
self._seed = int(time.time())
self.backend.random.set_seed(self._seed)
```